robot/rover/MobilePlatform/commands.h

```diff
@@ -35,9 +35,16 @@ class Commands {
     String ttoOff_cmd = "tto-off";
     String status_cmd = "status";
     String stop_cmd = "stop";
+    String ledOn_cmd = "indicator-on";
+    String ledOff_cmd = "indicator-off";
+    String ledManCtrl_cmd = "indicator-manual";
+    String ledAutoCtrl_cmd = "indicator-autonomous";
+    String ledEureka_cmd = "indicator-eureka";
 
     elapsedMillis sinceThrottle; // timer for reading battery, gps and imu data
 
+    uint8_t ledColor[3] = {0,0,0};
+    bool ledStatus = false;
```

Comment:
Fixed

Reply:
Sorry this only came to me afterwards but I just remembered that we also control the Teensy's builtin LED in our MCU code. Maybe instead of using the word LED for ledColor, ledStatus, ledOn() etc it would be better to just use the word 'indicator' because then it's less ambiguous

Reply:
Okay, will update with those changes.